src/web/styles.scss

```diff
@@ -358,6 +358,15 @@ a,
   padding-left: 15px;
 }
 
+.row {
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.padding-top-10px {
+  padding-top: 10px;
```

Comment:
Bootstrap has the `pt-10` notation for [spacing](https://getbootstrap.com/docs/5.0/utilities/spacing/), let's use bootstrap notations as much as possible.

Also, instead of adding a padding of 10 to every `main` class in the separate pages, it would be better to adjust the padding in `tm-page` so that all of the pages automatically have the same padding. In `page.component.scss`, padding is currently set to `0 0` for `main-content` for widths below 768px. Let's change it to `10px 5px`.